### ISSUE TITLE
Recommend removal of micro-dash

### ISSUE THREAD

**Issue by jmichaud-ips** [simontonsoftware/s-ng-utils#1]
I got tripped up on the use of micro-dash. Got more compile errors in my angular-cli 5.2 project once I added micro-dash. Plus we already use lodash.

**Comment by ersimont** [comment]
If you are running angular 5.2, I assume you must be on an older version of typescript. There wasn't a version 5.2 for the angular CLI, but version 1.7 installed Angular 5.2, probably that's what you're using. Looks like that uses Typescipt 2.5.

This library is currently requires Angular 7 and Typescript 3.1 - you can see that in the `package.json` file. I'm guessing that is the source of your problem. The current version of Microdash also requires version 3.1.

**Comment by jmichaud-ips** [comment]
yes you are right, Angular-cli 1.7 with Angular 5.2. Thanks for the follow up! nice code by the way.
